docs/build_arm.md

```diff
@@ -1,43 +1,60 @@
-# Building Larq Compute Engine for ARM-based boards
-This page descibes how to build a Larq Compute Engine (LCE) inference binary
-for 32-bit, as well as 64-bit ARM-based boards.
-[Bazel](https://bazel.build/) is the primary build
-system for LCE. However, since in some occasions Make build system is a more
-convenient build solution, we also provide scripts to build LCE with Make.
-
-In this guide, we use the [LCE benchmark tool](../larq_compute_engine/tflite/benchmark)
-source to build an inference binary.
-See [here](./inference.md) to find out how you can create your own
-custom LCE inference application.
-
-## Cross-compiling LCE with Bazel ##
-To cross-compile an LCE inference program for the ARM64 architecture,
-the bazel target needs to be built with the ```--config=aarch64``` flag.
-For example, to build the LCE benchmark tool,
-run the following command from the LCE root directory:
+# Building Larq Compute Engine for ARM-based systems
+This page descibes how to build a Larq Compute Engine (LCE) inference binary for 32-bit, as well as 64-bit ARM-based systems.
+You can either build LCE locally on your ARM device or cross-compile it from a non-ARM laptop or desktop computer.
+[Bazel](https://bazel.build/) is the primary build system for LCE, and it can be used to cross-compile a binary for ARM-based systems using a non-ARM host machine.
+To build on an ARM system itself we provide a script that uses the Makefile build system, because Bazel it is currently not available natively on the ARM architecture.
+
+This leaves us with three ways to build LCE, which we recommend in the following order:
+1. To cross-compile LCE from a host machine, see the section [Cross-compiling LCE with Bazel](#cross-compiling-lce-with-bazel)
+2. To build LCE on the ARM system itself, see the section [Building LCE with Make](#building-lce-with-make).
+3. It is also possible to cross-compile LCE using the Make system for users that do not whish to install Bazel, see the section [Cross-compiling LCE with Make](#cross-compiling-lce-with-make).
```

Comment:
to be consistent with the rest of the bullet points:
- To cross-compile LCE using the make ...